site/docs/types/1/0150-feedback.md

```diff
@@ -0,0 +1,20 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# 0150 Feedback
+
+## Feedback
+
+An important principle of good metadata it to be continually capturing the experience of subject matter experts. The *feedback* model captures comments and ratings from subject matter experts.
+
+![UML](0150-feedback.svg)
+
+Comments and ratings are a key mechanism for providing feedback on the metadata definitions by any user.  
+
+*Note:* that because comments inherit from [Referenceable](/egeria-docs/types/0/0010-base-model/#referenceable) they can be tagged, rated and commented on.
```

Comment:
Should probably use an admonition here:

```
!!! tip "Tag, rate, comment"
   Because comments inherit from [`Referenceable`](/egeria-docs/types/0/0010-base-model/#referenceable) they can be tagged, rated and commented on.
```

Reply:
Updated.